src/pageLayout/containers/TreeNav.tsx

```diff
@@ -72,6 +72,11 @@ const TreeSidebar: FC<ReduxProps & RouteComponentProps> = () =>
       }
     }
 
+    const handleEventing = (): void => {
+      const currentPage = location.pathname
+      event('help.and.support.clicked', {from: currentPage})
+    }
```

Comment:
what is the value for `currentPage` in this call?

Reply:
The whole output of location.pathname makes sense to me. But for the call to `event` - do we want 'from' to be a tag / meta, or a field? @ChitlangeSahas I think on my PRs for the onboarding eventing, we left the `meta`/`tag` object empty, and put this information in `values`.

`export const event = (
  measurement: string,
  meta: PointTags = {},
  values: PointFields = {}
): void `

Reply:
We'll probably want multiple event calls here, so we can separate the different clicks. We'll want one for clicking documentation, clicking faqs, and clicking community, since those are all different actions

Reply:
agreed with Bill's assessment, these should probably be values. This object should be moved to the third argument of `event` and the second argument should be an empty object literal `{}`